Cargo.toml

```diff
@@ -10,12 +10,18 @@ members = [
     "crates/client",
     "crates/hash",
     "crates/jose",
+    "crates/server",
     "crates/type",
 ]
 
 [dependencies]
 # Internal dependencies
-drawbridge-app = { path = "./crates/app" }
+drawbridge-byte = { path = "./crates/byte" }
```

Comment:
You'll need to add version specifiers for these in order to successfully publish.

Reply:
there's another PR coming for that